ExilenceNextApp/public/i18n/en/common.json

```diff
@@ -22,6 +22,7 @@
   },
   "title": {
     "login": "Login",
+    "redirect_not_working": "If the button doesn't work, copy the link below",
```

Comment:
`and paste it to your preferred browser` - Wouldn't be surprised to see someone asking what to do with this link